include/openspace/rendering/abufferrenderer.h

```diff
@@ -133,8 +143,13 @@ class ABufferRenderer : public Renderer, public RaycasterListener {
     GLuint _vertexPositionBuffer;
     int _nAaSamples;
 
+    float _hdrExposure;
+    float _hdrBackground;
+    float _gamma;
     float _blackoutFactor;
 
+    double * _mSAAPattern;
```

Comment:
Who owns the memory?  I couldn't find a new/delete in the cpp file.   I think it would be better to store it as `std::vector<double>` if it needs to be stored at all

Reply:
It is created when saving the framebuffer memory back to the array. The delete was missing. Fixed.